### PR TITLE
Target to use about 25MB for the new layered txpool by default

### PR DESCRIPTION
**Pull request by fab-10**
## PR description

Reduce the default memory usage of the txpool, to limit the added amount of memory used by Besu.
There 2 memory limited layers, ready and sparse, so to target 25MB we set the limit for a layer to 12.5MB

### CODE REVIEW

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPoolConfiguration.java [hunk 1]
```diff
@@ -64,7 +64,7 @@ enum Implementation {
   boolean DEFAULT_ENABLE_SAVE_RESTORE = false;
 
   File DEFAULT_SAVE_FILE = new File(DEFAULT_SAVE_FILE_NAME);
-  long DEFAULT_PENDING_TRANSACTIONS_LAYER_MAX_CAPACITY_BYTES = 50_000_000L;
+  long DEFAULT_PENDING_TRANSACTIONS_LAYER_MAX_CAPACITY_BYTES = 12_500_000L;
```
Comment (siladu) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPoolConfiguration.java:67]: Should this be 25MB or is there a non-obvious doubling going on somewhere?

Comment (fab-10) [reply ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPoolConfiguration.java:67]: Updated the description:

> There 2 memory limited layers, ready and sparse, so to target 25MB we set the limit for a layer to 12.5MB

**Review by siladu — APPROVED** [review]
Reckon this warrants a changelog entry?

**Comment by fab-10** [comment]
waiting for more metrics before merging

**Comment by fab-10** [comment]
> Reckon this warrants a changelog entry?

Existing changelog entry about the layered txpool, updated

**Pull request merged**